components/bl-popover/component.json

```diff
@@ -34,23 +34,26 @@
           "label": "Left"
         }
       ],
-      "handlerDescription": "This is a handler for the logic to determine the position of the popover relative to the button"
+      "handlerDescription": "This is a handler for the logic to determine the position of the popover relative to the button",
+      "settingTooltip": "Allows to determine the position of the popover relative to the button"
     },
     {
       "type": "number",
       "name": "delayMouseOver",
       "label": "Delay Mouse Over",
       "showInSettings": true,
       "defaultValue": "0",
-      "hasLogicHandler": false
+      "hasLogicHandler": false,
+      "settingTooltip": "Allows to determine the delay for On Mouse Over event"
     },
     {
       "type": "number",
       "name": "delayMouseOut",
       "label": "Delay Mouse Out",
       "showInSettings": true,
       "defaultValue": "0",
-      "hasLogicHandler": false
+      "hasLogicHandler": false,
+      "settingTooltip": "Allows to determine the delay for On Mouse Out event"
```

Comment:
We could simplify it using expressions like "Enter Delay" and "Exit Delay"

Reply:
I'm not only about tooltips, but about labels and property name too

Also we should rename `Tooltip.js` file -> `tooltip.js`

And update example screenshots, that haven't this two handlers.